### PR TITLE
GIF Search

### PR DESCRIPTION
**Pull request by beratasici**
I created a GIF Search Engine that uses [GIPHY API](https://developers.giphy.com/docs/api).

It needs user's own `API KEY` to function properly.

Check comments in `index.html` file for more information.

### CODE REVIEW

File: gifSearch/ajax.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+var ajax = {};
+ajax.x = function () {
+    if (typeof XMLHttpRequest !== 'undefined') {
+        return new XMLHttpRequest();
+    }
+    var versions = [
```
Comment (Neha) [inline gifSearch/ajax.js]: Wanted to know why we are checking all the XMLHttp till XMLhttp 2? (I am not aware of this. Hence, asking)

File: gifSearch/ajax.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        "MSXML2.XmlHttp.6.0",
+        "MSXML2.XmlHttp.5.0",
+        "MSXML2.XmlHttp.4.0",
+        "MSXML2.XmlHttp.3.0",
+        "MSXML2.XmlHttp.2.0",
+        "Microsoft.XmlHttp"
+    ];
+
+    var xhr;
+    for (var i = 0; i < versions.length; i++) {
```
Comment (Neha) [inline gifSearch/ajax.js]: To avoid the Javascript scope issue use _let i = 0_;

File: gifSearch/ajax.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    if (async === undefined) {
+        async = true;
+    }
+    var x = ajax.x();
+    x.open(method, url, async);
+    x.onreadystatechange = function () {
+        if (x.readyState == 4) {
+
+            if(x.status == 429) {
+                alert_popup(JSON.parse(x.responseText).message, 2);
```
Comment (Neha) [inline gifSearch/ajax.js]: Where is this function declared? _alert_popup_

File: gifSearch/index.html [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+    <style>
+
+        * {
+            margin: 0;
+            padding: 0;
+            box-sizing: border-box;
+            font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, sans-serif;
+        }
+
+        body {
```
Comment (Neha) [inline gifSearch/index.html:18]: Color code should be lower-case or uppercase to have consistency. I prefer lower-case.

**Review by Neha — CHANGES_REQUESTED** [review]
Thank you for taking time to create the code.

I am still reviewing the index.html.
Provided some small feedback on the rest files. Please review.

File: gifSearch/ajax.js (hunk 1 above)
Comment (beratasici) [reply gifSearch/ajax.js]: It was for IE support, now they're all obsolete.

**Comment by Neha** [comment]
no response. Hence, closing it

**Comment by Neha** [comment]
no response. Hence, closing it

**Pull request closed without merging**